handbook/brand.md

```diff
@@ -393,6 +393,119 @@ When conducting an incident post-mortem, answer the following three questions:
 2. Root Cause: Why did this error happen?
 3. Side effects: did this error have any side effects? e.g., did it corrupt any data? Did code that was supposed to run afterward and “finish something up” not run, and did it leave anything in the database or other systems in a broken state requiring repair? This typically involves checking the line in the source code that threw the error.
 
+# Article formatting guidelines
+
+To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in <ins><strong>Markdown</strong></ins>.
```

Comment:
I'm assuming that "<ins><strong>Markdown</strong></ins>" is destined to be a link.

Reply:
Yes, this will be updated to link to your Markdown guide. Please add "(TODO)" after it for now.